modular-docs-manual/content/topics/module_guidelines-concept.adoc

```diff
@@ -1,42 +1,33 @@
+When planning a concept module, look at nouns in related procedure modules and assemblies to find the concepts to explain to users. Explain only things that are visible to users. Even if a concept is interesting, it probably does not require explanation if it is not visible to users.
 
 image::concept-diagram.png[]
 
-// [bhardest] - I took a stab at reorganizing the descriptive content below.
+[id='concept-writing-the-introduction']
+.Writing the Introduction
 
-.Guidelines for Writing Concepts
-* Look at nouns and noun phrases in related procedure modules and user story assemblies to find the concepts to explain to users.
+The introduction to a concept module is a single, concise paragraph that provides a short overview of the module. A short description makes the module more usable, because users can quickly determine whether the concept is useful without having to read the entire module.
 
-* A concept module in product documentation should explain only things that are visible to users.
+The introduction typically answers the following questions:
 
-* If a product concept is interesting, but not visible to users, the concept probably does not require explanation in a concept module.
+* What is the concept?
+* Why should I care about the concept?
 
-* A concept module should NOT include numbered steps or other wording that instructs a user to execute a command or perform an action. Instead, put that information in a separate procedure module or user story assembly.
+[id='concept-writing-the-concept-explanation']
+.Writing the Concept Explanation
 
-.Writing the Short Description
-A concept module should start with a short description - a single, concise paragraph that provides a short overview of the module, its content, and why users should care about it. You can think of the short description as the module's thesis statement.
+The concept explanation describes the subject of the concept module.
 
-Writing a good short description makes your content more usable, as users can quickly determine whether the content is useful for their goals without having to read the entire concept module.
+Apart from paragraphs, you can use other AsciiDoc elements, such as lists, tables, or examples. Consider including graphics or diagrams to speed up the understanding of the concept.
 
-A good short description should answer the following questions:
-* What is this?
-* Why should I care about this?
+Do not include any instructions to perform an action, such as executing a command. Action items belong in procedure modules. See also link:http://www.informationmapping.com/fspro2013-tutorial/infotypes/infotype2.html[The Six Information Types] at _informationmapping.com_ for ways to present different types of conceptual information: principle, concept, structure, process, fact.
```

Comment:
I would argue against linking to outside sources from here because I find it distracting. Do I need to read the external source to understand the issue, or can I ignore the link? Can we put it in 'Additional Resources'?

Reply:
I always try to consider *when* the user could appreciate the link.

For resources that are needed after you read the whole module (additional reading, next steps, related modules), AR is the best place. But when I have a resource that is relevant to a specific place of the module (like here - the information types are needed when you are writing the concept description, which is described at a specific place in the module), I prefer to link right away.

Does that make sense?

Reply:
It makes perfect sense, and I agree. I just didn't think this was the case. But really, this is just nitpicking. The text is fine.